lua/fast_addons/playernick.lua

```diff
@@ -60,7 +60,7 @@ end
 
 function META:SetNick(nick)
 	nick = nick or self:RealNick()
-	if #string.Trim(string.gsub(nick,"%^%d","")) == 0 then
+	if #string.Trim(nick,string.gsub(nick,"%^%d","")) == 0 then
```

Comment:
`string.gsub` returns more than 1 argument and `string.Trim` started to accept more than one argument since some update.

What you want is `string.Trim((string.gsub(nick,"%^%d","")))` where the extra parenthesis around `string.gsub` makes it only select the first argument from gsub